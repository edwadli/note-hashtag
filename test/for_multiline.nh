for n in { "3" "2" "1" } do (
  Print "poo "
  PrintEndline n
)
for i in { 1 (1 + 1) (1 + 1 + 1) } do (
  Print "poo "
  PrintEndline (StringOfInt i)
)
