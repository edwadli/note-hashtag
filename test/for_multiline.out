poo 3
poo 2
poo 1
poo 1
poo 2
poo 3
